efficientdet/efficientdet_arch.py

```diff
@@ -505,34 +476,43 @@ def build_feature_network(features, config):
   return new_feats
 
 
-def bifpn_sum_config():
+def bifpn_config(weight_method='fastattn',quad_method='fastattn'):
   """BiFPN config with sum."""
   p = hparams_config.Config()
+  p.weight_method = weight_method or 'fastattn'
+  p.quad_method = quad_method or 'fastattn'
   p.nodes = [
-      {'feat_level': 6, 'inputs_offsets': [3, 4]},
-      {'feat_level': 5, 'inputs_offsets': [2, 5]},
-      {'feat_level': 4, 'inputs_offsets': [1, 6]},
-      {'feat_level': 3, 'inputs_offsets': [0, 7]},
-      {'feat_level': 4, 'inputs_offsets': [1, 7, 8]},
-      {'feat_level': 5, 'inputs_offsets': [2, 6, 9]},
-      {'feat_level': 6, 'inputs_offsets': [3, 5, 10]},
-      {'feat_level': 7, 'inputs_offsets': [4, 11]},
+      {'feat_level': 6, 'inputs_offsets': [3, 4] , 'weight_method': p.weight_method},
```

Comment:
Is this the main change? So now each layer has four paths: 

(top down + bottom up)  + (bottom_up + top_down)

Any reason to have  two different weight_methods?

Reply:
![image](https://user-images.githubusercontent.com/43991882/87130173-1c1e4580-c2b0-11ea-9ad9-d0e026bd73e5.png)

Yea, basically creating this structure. At first I used fast normalized fusion for the entire FPN , but then I experimented a bit and noticed the network performs even better when you use unweighted connections in the two parallel FPNs and weighted connection for fusion of the two. This gave higher accuracy and better stability. I didn't get to experimenting this variant with heavier models though. So still requires training and testing.